create-release.py

```diff
@@ -442,11 +449,40 @@ def prepare_extensions_release() -> None:
     print("-----------------------------------------------------------------")
 
     extensions = {
-        "elyra-code-snippet-extension": ["code-snippet-extension", "metadata-extension", "theme-extension"],
-        "elyra-code-viewer-extension": ["code-viewer-extension"],
-        "elyra-pipeline-editor-extension": ["pipeline-editor-extension", "metadata-extension", "theme-extension"],
-        "elyra-python-editor-extension": ["python-editor-extension", "metadata-extension", "theme-extension"],
-        "elyra-r-editor-extension": ["r-editor-extension", "metadata-extension", "theme-extension"],
+        "elyra-code-snippet-extension": SimpleNamespace(
+            packages=["code-snippet-extension", "metadata-extension", "theme-extension"],
+            description=f"The Code Snippet editor extension adds support for reusable code fragments, "
+            f"making programming in JupyterLab more efficient by reducing repetitive work. "
+            f"See https://elyra.readthedocs.io/en/{config.new_version}/user_guide/code-snippets.html",
+        ),
+        "elyra-code-viewer-extension": SimpleNamespace(
+            packages=["code-viewer-extension"],
+            description="The Code Viewer extension adds the ability to display a given chunk of code "
+            "(string) in a transient read-only 'editor' without needing to create a file."
+            "This extension will be available in JupyterLab core in a near future release and removed "
+            "from Elyra as a standalone extension.",
+        ),
+        "elyra-pipeline-editor-extension": SimpleNamespace(
+            packages=["pipeline-editor-extension", "metadata-extension", "theme-extension"],
```

Comment:
This should require `elyra-code-viewer-extension`? afaik the editor is the only consumer of that functionality.